src/tests/test_configs/yaml/bluebutton-sample-config-missing-callback.yaml

```diff
@@ -0,0 +1,11 @@
+# BB2 target service SANDBOX or PRODUCTION
```

Comment:
Because you created generic code, you could actually decrease the number of test files you have here. Basically you want to make sure that what gets loaded in via JSON and what gets loaded in via YAML is the same. Once you have verified that in a test, you can just do all the edge case testing in one (say JSON). You don't need to duplicate them all for YAML as well.

Reply:
@nbragdon I like this!  Will update the tests to do this.  TY!

Reply:
@nbragdon Updated the tests and removed  .yaml files for this. Left one YAML file for the JSON <-> YAML read_config() test.

This is much cleaner! TY!